### PR TITLE
Feature/rangle/statistic card

### PR DESCRIPTION
**Pull request by nataliya-ioffe**
# Description

Implement initial Statistic Card + Grid

For mobile, the 'grid' experience is actually leveraging our Carousel component per design feedback from WV.  The Carousel was adjusted to be able to take in any kind of card. In the storybook there's now 2 variants shown for Carousel -- 1 with program cards, 1 with statistic cards.

For desktop, we leverage CSS Grid for the "Statistic Grid."

Additionally, TSX is removed from any file I worked on and switched to JSX.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ x ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

# Checklist:

- [ ] My code follows the style guidelines of this project at all breakpoints
- [ ] This matches all the required acceptance criteria
- [ ] This code has been tested in Chrome, Firefox & Safari

### CODE REVIEW

File: wvc-ourwork-nextjs/components/ProgramCard/ProgramCard.stories.tsx [hunk 1]
```diff
@@ -1,23 +1,23 @@
-import React from 'react';
-import { ComponentStory, ComponentMeta } from '@storybook/react';
+import React from 'react'
```
Comment (scottfuller4) [inline wvc-ourwork-nextjs/components/ProgramCard/ProgramCard.stories.tsx:1]: Should this file be deleted now that there's also a jsx version?

File: wvc-ourwork-nextjs/components/StatisticCardGrid/StatisticCardGrid.scss [hunk 2]
```diff
@@ -0,0 +1,45 @@
+@import '../../styles/lumen-tokens';
+
+.statistic-card {
+  background-color: $white;
+  border-radius: 2px;
+  box-shadow: 2px 2px 16px rgba(0, 0, 0, 0.05);
```
Comment (scottfuller4) [inline wvc-ourwork-nextjs/components/StatisticCardGrid/StatisticCardGrid.scss:6]: Let's continue using rem moving forward!

File: yarn.lock [hunk 3]
```diff
@@ -0,0 +1,4 @@
+# THIS IS AN AUTOGENERATED FILE. DO NOT EDIT THIS FILE DIRECTLY.
```
Comment (scottfuller4) [inline yarn.lock]: Can you delete this file? There should only be a yarn.lock within `wvc-ourwork-nextjs/`

File: wvc-ourwork-nextjs/components/ProgramCard/ProgramCard.stories.tsx (hunk 1 above)
Comment (scottfuller4) [reply wvc-ourwork-nextjs/components/ProgramCard/ProgramCard.stories.tsx:1]: Or is this supposed to be the story for the StatisticCard?

**Review by scottfuller4 — COMMENTED** [review]
Overall, looking really good! Just a couple minor cleanup things:

- It looks like there isnt a story for the statistic card but there are two for the program cards
- Within Carousel.jsx can you remove the `ally` prop on line 45 - 47? It's adding extra text below the carousel which we don't want - I think this was my bad from when I first built it, but figure we should remove it now
- On the carousel docs page, if you click the next button it moves both carousels. Could you take a look into making them act independently now that we're using it for multiple types of cards?

Let me know if you have any qs! :)

File: wvc-ourwork-nextjs/components/ProgramCard/ProgramCard.stories.tsx (hunk 1 above)
Comment (nataliya-ioffe) [reply wvc-ourwork-nextjs/components/ProgramCard/ProgramCard.stories.tsx:1]: Think something weird happened with the rebase. Ill fix it.

File: wvc-ourwork-nextjs/components/StatisticCardGrid/StatisticCardGrid.scss (hunk 2 above)
Comment (nataliya-ioffe) [reply wvc-ourwork-nextjs/components/StatisticCardGrid/StatisticCardGrid.scss:6]: From my experience box-shadow is more often written with pixels due to how small the values are. Same with something like border-radius. (lumen token border radius also written in pixel) I'll leave this for now but we can change later if needed

File: yarn.lock (hunk 3 above)
Comment (nataliya-ioffe) [reply yarn.lock]: Ty!

**Review by scottfuller4 — APPROVED** [review]
All looks good, could you just add a single Statistic Card story before merging? 🚢

**Pull request merged**